bibliopixel/animation/collection.py

```diff
@@ -1,11 +1,13 @@
 from .. project import aliases, project
 from . import animation, runner
+import traceback
+from .. import log
 
 
 class Collection(animation.BaseAnimation):
-    def __init__(self, layout, animations=None):
+    def __init__(self, layout, animations=None, no_fail=False):
```

Comment:
:) I debated about that... but if you were running this from a script directly and one of your animations in a sequence failed, then if you missed the traceback printed out to the console you'd probably wonder what happened to that one animation. That was my only concern, but I could go either way on it.